cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -1,3 +1,19 @@
+{% set cluster_state = 'new' -%}
+{% set host_name = pillar.get('master_name', '') -%}
+{% set etcd_cluster_array = pillar.get('initial_etcd_cluster', '').split(',') -%}
+{% set etcd_cluster = '' -%}
+{% set vars = {'etcd_cluster': '', 'cluster_state': cluster_state} -%}
+{% for host in etcd_cluster_array -%}
+  {% if etcd_cluster != '' -%}
+    {% set cluster_state = 'existing' -%}
```

Comment:
`cluster_state` **is** **set** to `new` if initial etcd cluster has 1 element. In case of one elelement, we set `etcd_cluster` after this comparision.